src/Test/Tasty/Plutarch/Property.hs

```diff
@@ -102,6 +105,38 @@ peqProperty expected gen shr comp =
                 Left e -> unexpectedError e
                 Right s' -> sameAsExpected s'
 
+{- | `alwaysfailproperty` universially checks if given computation fails.
+ This function operates very similarly to `peqProperty` but just checking
+ for the failures. Since output does not need to be checked, `PEq` instance
+ is not required.
```

Comment:
This probably needs elaboration: specify that we check for _crashes_ or _errors_, rather than anything else.